sql/src/test/java/io/crate/integrationtests/InsertIntoIntegrationTest.java

```diff
@@ -317,6 +318,26 @@ public void testInsertWithPrimaryKey() throws Exception {
     }
 
     @Test
+    public void testInsertWithPrimaryKeyFailing() throws Exception {
+        this.setup.createTestTableWithPrimaryKey();
+
+        Object[] args = new Object[]{"1",
+            "A towel is about the most massively useful thing an interstellar hitch hiker can have."};
+        execute("insert into test (pk_col, message) values (?, ?)", args);
+        refresh();
+
+        expectedException.expect(SQLActionException.class);
+        expectedException.expectMessage("DuplicateKeyException: A document with the same primary key exists already");
+        args = new Object[]{"1",
+            "I always thought something was fundamentally wrong with the universe."};
+        execute("insert into test (pk_col, message) values (?, ?)", args);
+        refresh();
+
+        GetResponse response = client().prepareGet("test", "default", "1").execute().actionGet();
```

Comment:
pls don't use ES api here, no need to do so. also afaik its not needed to validate that the inserto did not ocurre